draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -318,7 +318,7 @@
    after message parsing has delineated the individual field lines.
 </t>
 <t>
-   Although the line terminator for the start-line and header
+   Although the line terminator for the start-line and
    fields is the sequence CRLF, a recipient &MAY; recognize a
```

Comment:
Here, "header" is correct, no?

Reply:
I think that's correct. Could you please make suggestions using the +- icon above ^^ (cmd-g shortcut, apparently)?